blazar_dashboard/api/client.py

```diff
@@ -391,3 +392,13 @@ def extra_capability_values(request, name):
     rows += dictfetchall(cursor)
 
     return rows
+
+
+def get_floatingip_network_id(request, network_name):
+    """Return default network id for floatingip reservation"""
+    # Use case-agnostic check. DEV@TACC uses Public instead of public as net.
```

Comment:
Just had a thought: what if, instead of doing this, we allow a regex to be the value of the `network_name` configuration option? I think assuming case-insensitive is going to be controversial if we ever want to merge this upstream. It would also remove the need for the Chameleon-specific comment here (comment would move to the blazar site-configuration, where it makes more sense.)

Reply:
Updated to use network_name_regex. This would be added to node_custom_configuration with the corresponding regex as `[pP]ublic` for our purposes.